README.md

```diff
@@ -1,2 +1,73 @@
-# prometheus-opensearch-dashboards-exporter
+# Prometheus Opensearch Dashboards Exporter
 An exporter for OpenSearch Dashboards
+
+
+## Installing
+With [poetry](https://python-poetry.org/) installed, clone this repo and run:
+
+```shell
+poetry install
+```
+
+```shell
+poetry build
```

Comment:
Is this even needed?  I can already call "prometheus-opensearch-dashboards-exporter" immediately after "poetry install", and this is in a fresh venv I created for Poetry 2.0.